13LessonThirteen/homework/css/style.css

```diff
@@ -0,0 +1,88 @@
+body {
+  font-family: Roboto Mono,Helvetica Neue,sans-serif;
+}
+
+.hero {
+  height: 450px;
+  position: relative;
+  background-image: url("https://hd.unsplash.com/photo-1449247709967-d4461a6a6103");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.hero .col {
+  position: absolute; z-index: 1; top: 0; left: 0;
+}
+
+.small-img {
+  width: 350px;
+  height: 250px;
+  padding: 0 20px 20px 0;
+}
+
+.large-img {
+  height: 480px;
+  width: 300px;
+}
+
+h1 {
```

Comment:
FYI this `h1` isn't centered in your media query. Maybe there's a way you can center it using `margin: 0 auto`?